5gms/5G_APIs-overrides/TS26512_CommonData.yaml

```diff
@@ -0,0 +1,244 @@
+openapi: 3.0.0
+info:
+  title: 5GMS Common Data Types
+  version: 2.0.1
```

Comment:
Should probably make this "2.Y.Z" to indicate that this is a pre-standardisation proposal.